addon/components/bs-form/group.js

```diff
@@ -61,6 +58,7 @@ export default class FormGroup extends Component {
    * @type string
    * @public
    */
+  @localCopy('args.validation') validation;
```

Comment:
We use `@localCopy` because we know that *subclasses* (like that from `e-b-c-v`) will need to mutate that (given our current validation API), right? Have you tested `e-b-c-v` using this branch, so we know this *really* works?

Reply:
To be honest I have no idea why I used `@localCopy` decorator here. It doesn't seem to be needed at all. I didn't found any place in which it's overwritten locally. Neither in ember-bootstrap itself nor in ember-bootstrap-changeset-validations nor in ember-bootstrap-cp-validations. I guess it's just a leftover from an early iteration? I removed it in c8d51ffdc662d1aecbdc531011972540b9242d9a.

> Have you tested e-b-c-v using this branch, so we know this really works?

Not sure if `e-b-c-v` refers to `ember-bootstrap-changeset-validations` or `ember-bootstrap-cp-validations`. I have tested with the first one. Pull request showing required changes to ember-bootstrap-changeset-validations and passing CI pipeline could be found here: https://github.com/kaliber5/ember-bootstrap-changeset-validations/pull/34/files I haven't tested with ember-bootstrap-cp-validations.

Reply:
> Not sure if e-b-c-v refers to ember-bootstrap-changeset-validations or ember-bootstrap-cp-validations. 

To the former, indeed! 😆. So all good...